invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookTab/components/BookAccordion/BookAccordion.js

```diff
@@ -26,7 +26,7 @@ export default class BookAccordion extends Component {
           Abstract
         </Accordion.Title>
         <Accordion.Content active={activeIndex === 0}>
-          <p>{this.props.data.abstracts[0]}</p>
+          <p>{data.metadata.abstracts[0]}</p>
```

Comment:
Shouldn't we iterate on abstracts instead of just displaying the first?

Reply:
for this schema is not finished yet, I'm not sure that's the final form